gusto/diagnostics.py

```diff
@@ -66,6 +95,21 @@ def compute(self, state):
         return self.field(state.mesh).project(sqrt(dot(u, u))/sqrt(self.area(state.mesh))*dt)
 
 
+class HorizontalVelocity(DiagnosticField):
+    name = "HorizontalVelocity"
+
+    def field(self, mesh):
+        if hasattr(self, "_field"):
+            return self._field
+        self._field = Function(FunctionSpace(mesh, "CG", 1), name=self.name)
+        return self._field
+
+    def compute(self, state):
+        u = state.fields("u")
+        uh = u[0]
+        return self.field(state.mesh).interpolate(uh)
```

Comment:
Renamed it VelocityX to avoid confusion. Similarly, Meridional and Vertical velocities are renamed as VelocityY and VelocityZ. KineticEnergyV is renamed as KineticEnergyY.